homework/Homework01Tolstoy.java

```diff
@@ -0,0 +1,41 @@
+package homework;
```

Comment:
this line threw me for a loop-- could have used some documentation on running the script